backend/src/identity/oidc/oidc.service.ts

```diff
@@ -166,15 +177,18 @@ export class OidcService {
     const oidcConfig = clientConfig.config;
     const params = client.callbackParams(request);
     const code = request.session.oidcLoginCode;
+    const state = request.session.oidcLoginState;
```

Comment:
While it isn't explicit necessary since the state is regenerated for every OIDC login, it would still be better to clean it up after a finished login like with the PKCE `code`.

The clean-up takes place in this same method at the end by simply setting the session value to undefined.

Reply:
Thank you very much for your patience, I added the code to clean up the state.

https://github.com/hedgedoc/hedgedoc/pull/5831/files#diff-da64bf255fae053e7438a387a3b1a178a495330d35b73fa1d8272f467aba180dR234-R235